code/src/components/app-range-slider/AppRangeSlider.types.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeEvent } from "react";
+
+import { SliderProps } from "@mui/material/Slider";
+
+type AppRangeClassNames = {
+  root: string;
+  range: string;
+  toolbar: string;
+  toolbarInput: string;
+};
```

Comment:
what do you think.
If all values of keys are `string`
can we improve it ?

Reply:
Refactored as follows:

```tsx
type AppRangeClassName = "root" | "range" | "toolbar" | "toolbarInput";
type AppRangeClassNames = Partial<Record<AppRangeClassName, string>>;
```